base/linalg/lapack.jl

```diff
@@ -94,6 +100,17 @@ function chkfinite(A::StridedMatrix)
     return true
 end
 
+# LAPACK version number
+function laver()
+    major = BlasInt[0]
+    minor = BlasInt[0]
+    patch = BlasInt[0]
+    ccall((@blasfunc(ilaver_), liblapack), Void,
```

Comment:
```jl
julia> macro whatisthis(thing)
           return typeof(thing)
       end

julia> @whatisthis(:ilaver_)
Expr

julia> @whatisthis(ilaver_)
Symbol

julia> for s in (:ilaver_,)
           @eval println(@whatisthis($s))
       end
Symbol
```
I was also puzzled by this. The macro could be modified to work for `Expr` as well.